atomtopubsub.py

```diff
@@ -25,29 +49,52 @@ def setup_logging(level):
 
 
 # We feed the pubsub nodes
+
 async def parse(parsed, xmpp):
     imp.reload(config)
-
+    
     # We parse all the feeds
     for key, feed in config.feeds.items():
         print(colored('>> parsing %s' % key, 'magenta'))
-        f = feedparser.parse(feed['url'])
-
+        try:
+            f = feedparser.parse(feed['url'])
+        except Exception as e:
+            print(e)
+            continue    
+        version = f.version
+        print('Version %s' % version)
+        
+        
         if f.bozo == 1:
             print('XML Error')
             if hasattr(f.bozo_exception, 'getMessage'):
                 print(f.bozo_exception.getMessage())
             if hasattr(f.bozo_exception, 'getLineNumber'):
                 print('at line %s' % f.bozo_exception.getLineNumber())
-
+        
```

Comment:
Can you please ensure that you're not adding spaces where they are not needed :) ?